RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Automated releases
+TBD
+
+# Manual releases
+1. Set BUILD_VERSION, e.g. 1.0.0-beta.3. Ensure this tag exists for contiv-ui repo.
+
+	```
+	export BUILD_VERSION=<build version>
+	```
+
+2. Build docker binary image. This would create a docker image contiv/auth_proxy:$BUILD_VERSION.
+
+	```
+	make build
+	```
+2. Tag specific commit/HEAD with BUILD_VERSION. Push tag to master.
+
+	```
+	git tag $BUILD_VERSION
+	git push origin $BUILD_VERSION
+	```
+3. Execute scripts/release.sh. Creates a new release on GitHub.
+
+	```
+	export GITHUB_USER=contiv
+	export GITHUB_TOKEN=<your token here>
+	./scripts/release.sh
+	```
+4. Push image to docker hub.
```

Comment:
Do we want to keep this as a separate, manual step or can we fold it into the release script?